.github/workflows/workflow.yaml

```diff
@@ -43,6 +45,10 @@ jobs:
       - name: Get Stack snapshot install directory
         id: stack-snapshot
         run: |
+          # XXX Workaround
+          # https://github.com/commercialhaskell/stack/issues/5028.
+          mv ${{ matrix.stack_yaml }} stack.yaml || true
```

Comment:
Why would `mv` return non-zero exit code here?

Reply:
Because in one row of the matrix,

```
mv ${{ matrix.stack_yaml }} stack.yaml 
```

evaluates to

```
mv stack.yaml stack.yaml
```

Reply:
Rewrote this line to more verbose but more self-explanatory code.